core/commands/wallet.go

```diff
@@ -190,19 +204,27 @@ var walletPasswordCmd = &cmds.Command{
 	},
 	Options: []cmds.Option{},
 	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
-		cfg, err := cmdenv.GetConfig(env)
+		n, err := cmdenv.GetNode(env)
+		if err != nil {
+			return err
+		}
+		cfg, err := n.Repo.Config()
 		if err != nil {
 			return err
 		}
 		if cfg.UI.Wallet.Initialized {
-			return errors.New("Already init, cannot set pasword again.")
+			return errors.New("Already init, cannot set password again.")
 		}
-		n, err := cmdenv.GetNode(env)
+		cipherMnemonic, err := wallet.EncryptWithAES(req.Arguments[0], cfg.Identity.Mnemonic)
```

Comment:
not sure, seems req.Arguments[1] was not used here.

Reply:
There is only one argument.. req.Arguments[0] is the password